### ISSUE TITLE
Qt --convert my.ui

### ISSUE THREAD

**Issue by mottosso** [mottosso/Qt.py#131]
### Goal

In response to #99 and #129, provide a safe conversion from `.ui` files compiled with PySide2 to an equivalent Qt.py-compatible module.

### Implementation

From a plain-text Python module, produce a module with full cross-binding compatibility with Qt.py using plain old search-and-replace.

```bash
$ pyside2-uic.exe my_ui.ui -o my_ui.py
$ python -m Qt --convert my_ui.py  # save original as `original_my_ui.py`
$ cat my_ui.py
```

**Comment by mottosso** [comment]
Implemented in [0.5.0](https://github.com/mottosso/Qt.py/releases/tag/0.5.0).

**Comment by fredrikaverpil** [comment]
I guess that with PR https://github.com/mottosso/Qt.py/pull/142 merged it should now also be possible to implement `--compile`.
